src/Symfony/Component/Routing/Matcher/Dumper/StaticPrefixCollection.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Matcher\Dumper;
+
+/**
+ * Prefix tree of routes preserving routes order.
+ *
+ * @author Frank de Jonge <info@frankdejonge.nl>
+ *
+ * @internal
+ */
+class StaticPrefixCollection
+{
+    /**
+     * @var string
+     */
+    private $prefix;
+
+    /**
+     * @var array[]|StaticPrefixCollection[]
+     */
+    private $items = array();
+
+    /**
+     * @var int
+     */
+    private $matchStart = 0;
+
+    /**
+     * @param string $prefix
+     */
```

Comment:
This phpdoc can be removed. As a rule of thumb, we don't add phpdocs when it does not add any useful information.